### PR TITLE
Style edits

### PR DESCRIPTION
**Pull request by emileighoutlaw**
minor style edits

### CODE REVIEW

**Comment by mbland** [comment]
cc: @johnscancella

File: pages/build-promotion.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
...
-title: Build Promotion
+title: Build promotion
 ---
 <a name="build-promotion"></a>
-* Every code commit to version control creates a potentially deployable artifact in the Continuous integration server.
-Therefore a well defined and version controlled process for promoting a build from Dev(or continuous integration artifact)->Test->Staging->Prod needs to exist.
-* These built artifacts should not change when deploying to multiple environments(i.e. built once), and are stored in a artifact repository for easy retrieval. 
-This ensures that the built product is the same when testing in each environment, and that only configuration changes(if any) are different between environments.
+* Every code commit to version control creates a potentially deployable artifact in the continuous integration server.
+This being the case, we need a well defined and version controlled process for promoting a build (a continuous integration artifact) from development. ->Test->Staging->Prod.
```
Comment (mbland) [inline pages/build-promotion.md]: Perhaps the end should read "from development to test, staging, and eventually production."

Comment (johnscancella) [inline pages/build-promotion.md]: Agree with @mbland "from development to test, staging, and eventually production." sounds good

Comment (emileighoutlaw) [inline pages/build-promotion.md]: And we have a quorum! Made the change. :)

**Comment by johnscancella** [comment]
This makes me so happy! Can't wait to show this to my fellow developers

**Pull request merged**